src/ubuntu/manifest.json

```diff
@@ -476,6 +476,20 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "architecture": "arm",
+              "dockerfile": "src/ubuntu/24.10/helix",
+              "os": "linux",
+              "osVersion": "oracular",
+              "tags": {
+                "ubuntu-24.10-helix-arm32v7": {}
+              },
+              "variant": "v7"
+            }
+          ]
+        },
```

Comment:
Nit: to follow the existing convention, this should be listed after arm64.

Reply:
Oops. Yes.